src/main/webapp/app/course/learning-paths/learning-path-graph/learning-path-graph.component.ts

```diff
@@ -6,6 +6,11 @@ import { Subject } from 'rxjs';
 import { LearningPathService } from 'app/course/learning-paths/learning-path.service';
 import { NgxLearningPathDTO, NgxLearningPathNode } from 'app/entities/competency/learning-path.model';
 
+export enum LearningPathViewMode {
+    GRAPH,
+    PATH,
```

Comment:
Please NEVER use ts-enums without providing values, the default implementation is horrible.
```suggestion
    GRAPH = 'graph',
    PATH = 'path',
```

If you are wondering why: The default values for graph and path are 0 and 1:
```
private method(value: LearningPathViewMode | undefined) { 
    if (value) {
        return true;
    }
    return false;
}

method(GRAPH); // returns false since GRAPH is 0 and 0 is falsy
```